packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -125,7 +125,13 @@
         padding: 0 0.4rem;
         border-radius: $BORDER_RADIUS * 4;
         background-color: var(--text-loss-danger);
-        color: var(--status-colored-background);
+        @include mobile {
+            top: unset;
+            right: unset;
+            bottom: calc(100% - 7px);
+            left: 50%;
+            transform: translate(-50%, 0);
```

Comment:
`left 50%` and `transform` are needed for proper alignment (independent of the width, as it changed due to different translation)

Reply:
@kate-deriv does this not contradict `left: 50%`? can we just put `left: 0;` and not set `transform` at all? :)

Reply:
No, it didn't worked. 
```
left: 50%;
transform: translate(-50%, 0);
```
is used for centering absolute positioned element

Reply:
left 50% will move the element exactly at the center of the main container where this element belongs! BUT translateX(50%) will move the element right exactly to 50% of its width,and NOT at the center of the whole Container element!